nuxt.config.ts

```diff
@@ -21,6 +21,37 @@ export default defineNuxtConfig({
   app: {
     head: siteMeta,
   },
+  components: {
+    dirs: [
+      // Use this to find all the dirs `find . -name "*.vue" -exec dirname {} \; | sort -u`
+      { path: '~/components', sourcemap: true }, // Enable source map generation for components
+      { path: '~/pages', sourcemap: true }, // Enable source map generation for components
+      {
+        path: '~/node_modules/vue-instantsearch/src/components',
+        sourcemap: true,
+      }, // Enable source map generation for components
+      {
+        path: '~/node_modules/vite-plugin-vue-inspector/src',
+        sourcemap: true,
+      }, // Enable source map generation for components
+      {
+        path: '~/node_modules/@vite-pwa/nuxt/dist/runtime/components',
+        sourcemap: true,
+      }, // Enable source map generation for components
```

Comment:
qq, do we need the source maps for this node_modules?

Reply:
Ok the sourcemaps are good to have, but it turns out nuxt prepare is reading this section and for example "pages" will crash because the route names are not defined and will by default be defined as the concat of the path names which ends up using symbol names like 404 (which is a number) or [profileAddress]BuyLyx which is obviously not a good symbol name. I removed pages and some of the other one. Each component that could possibly cause errors at runtime is nice to have because when an error occurs then sentry will show the actual source code rather than just minified javascript garbage. But obviously none of this is like "oh my god, there are no symbols" :)